doc/examples.md

```diff
@@ -0,0 +1,108 @@
+# Examples: Getting Started using Padertorch
+
+Padertorch's main purpose is to provide an open interface that simplifies tedious work and works with as many different 
+workflows as possible. Due to the open design of Padertorch, there is no "right" or "wrong" way to train a neural network.
+However, to make getting started easier, we have provided several examples with our recommended workflow when using Padertorch. 
+
+We recommend the usage of the following components when using Padertorch:
+  - [sacred](): Experiment Organization / Ensuring reproducability
+  - [lazy_dataset](https://github.com/fgnt/lazy_dataset): Data preparation and on-the-fly data generation
+  - [Padertorch Configurable](): Adjustment of training hyperparameters
+  
+The examples mostly use all of these components, whereas the simple examples  (See [Getting Started]()) omit some of these components to reduce the overall complexity of the example as much as possible.
+
+
+## Basic Structure of a Training:
+
+Each example several has several steps:
+  - Experiment setup (directory creation, reproducability, logging)
+  - Data preprocessing
+  - Training
+  - (Optional:) Evaluation
+
+### Experiment setup
+During the experiment setup, the experiment directory is created. We recommend using [Sacred]() to monitor each training.
+Inside this directory, the checkpoints of the training as well as the event files for visualization are saved. 
+The examples using Padertorch Configurable and Sacred also save the model hyperparameters in this folder and log the 
+changes over multiple starts of the same training.
+
+In its most basic form this step looks as simple as:
+``` python
+import os
+
+storage_dir = pt.io.get_new_storage_dir('my_experiment')
```

Comment:
If promoting get_new_storage_dir you should mention that we expect STORAGE_ROOT to exist. Perhaps also mention the other possibility to scip get_new_storage_dir and directly use get_new_subdir